src/Core/src/Platform/Windows/ViewExtensions.cs

```diff
@@ -3,11 +3,18 @@
 using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Automation;
 using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
 
 namespace Microsoft.Maui
 {
 	public static class ViewExtensions
 	{
+		public static void TryMoveFocus(this FrameworkElement nativeView, FocusNavigationDirection direction)
+		{
+			if (nativeView?.XamlRoot?.Content is UIElement elem)
+				FocusManager.TryMoveFocus(direction, new FindNextElementOptions { SearchRoot = elem });
```

Comment:
https://github.com/microsoft/microsoft-ui-xaml/issues/3587#issuecomment-769428178

We now have to use the overloads that use XamlRoot since with WinUI there could be many roots - like multiple windows and xaml islands.